src/renderer/src/App.jsx

```diff
@@ -26,13 +26,15 @@ function App ({ path }) {
   const [showModalImportApplication, setShowModalImportApplication] = useState(false)
   const [showModalCreateApplication, setShowModalCreateApplication] = useState(false)
   const featureFlag = isDevMode()
+  const [showErrorComponent, setShowErrorComponent] = useState(false)
 
   const {
     ErrorBoundary,
-    didCatch,
+    // didCatch,
```

Comment:
Why commented? If useless, please remove it

Reply:
removed